### ISSUE TITLE
The quickstart in README.md only good for local copy of Buk, but won't work on Github

### ISSUE THREAD

**Issue by frnkly** [hang-up/buuk#25]
I'd make a PR, but I'm not sure what the best way to re-word the README would be. In a nutshell, the current guide will not allow you to use Buk for your own projects on Github for the 2 following reasons:

1. `git clone` will clone the _original_ version of Buk. Meaning that you won't be able to have your own copy of the repo on your Github account (the remote will point to the original version at https://github.com/hang-up/buuk.git)

This could be fixed by updating the remote locally after creating an empty repo on Github:
```shell
git remote set-url origin https://github.com/YOUR OWN USER NAME/buk.git
```
2. `npm install buk` installs the framework as a dependency, but does not copy the files into your current project.

For example, once the project is setup like so:
```shell
mkdir some-project && cd some-project
npm init
npm install buk && npm install
```
Running `npm run build` or `npm run dev` will result in the error:
```shell
npm ERR! missing script: build
```
(or missing script: dev) since no such script is defined in `package.json`. Even with files inside `docs/` and a manifest file, the scripts cannot run since they don't exist. I don't know enough about npm to suggest a fix for that one :/

In other words, you can run a local copy of Buk using `git clone`, but I'm not sure how you can then host your Buk docs on your own repo (unless you update the remote). As for using `npm install buuk` I think the README is missing a few steps..

I personally had to fork the repo to get the desired results.

**Comment by hang-up** [comment]
You're right, the readme might be a little bit too succinct. Thanks for the issue, I'll push an updated readme in the very next future!
